content/realtime/channels/channel-parameters/rewind.textile

```diff
@@ -0,0 +1,87 @@
+
+At most 100 messages will be sent in a rewind request. If the number of messages within the specified interval is greater than that limit, then only the most recent messages up to that limit are sent. The attachment succeeds, but truncation of the message backlog is indicated as a non-fatal error in the attachment response.
+
+By default, a maximum of two minutes of channel history is available when attaching. This means that a rewind time specifier of greater than two minutes will only be able to rewind by two minutes. If a channel has persistence enabled, then it is possible to rewind back in time by up to the persistence TTL on the channel.
+
+The channel position expressed by a @rewind@ parameter only has an effect on an initial channel attachment. Any subsequent reattachment of the same channel on the same connection, in order to resume the connection, will attempt to resume with continuity from the point at which the connection dropped. (There are a few exceptions to this: in particular, client libraries earlier than v1.2 that have been disconnected for over two minutes, and all clients when using "@recover@ mode":/realtime/connection#connection-state-recovery ; in both cases the previous attachment state is not preserved).
+
+Any @rewind@ parameter value that cannot be parsed either as a number or a time specifier will cause the attachment request to fail and return an error.
+
+h2(#examples). Examples
+
+h3(#rewind-example-ably). Rewind example with an Ably client library
+
+To subscribe to a channel, getting the most recent message if available:
+
+```[jsall]
+  // only with ably-js v1.2 or later
+  const realtime = new Ably.Realtime('{{API_KEY}}');
+  realtime.channels.get('{{RANDOM_CHANNEL_NAME}}', {
+    rewind: '1'
+  }).subscribe(msg => console.log("Received message: ", msg));
+```
+
+```[jsall]
+  // with ably-js v1.1 or below
+  const realtime = new Ably.Realtime('{{API_KEY}}');
+  const channel = realtime.channels.get('[?rewind=1]{{RANDOM_CHANNEL_NAME}}');
+  channel.subscribe(msg => console.log("Received message: ", msg));
+```
+
+h3(#rewind-example-sse). Rewind example with SSE
+
+To subscribe to a channel, getting the most recent message if available:
+
+```[javascript]
+  var querystring = 'v=1.2&channels={{RANDOM_CHANNEL_NAME}}&rewind=1&key={{API_KEY}}';
+  var eventSource = new EventSource('https://realtime.ably.io/event-stream?' + querystring);
+```
+
+h3(#rewind-examples-mqtt). Rewind example with MQTT
+
+```[nodejs]
+  var mqtt = require('mqtt');
+  var options = {
+    keepalive: 30,
+    username: 'FIRST_HALF_OF_API_KEY',
+    password: 'SECOND_HALF_OF_API_KEY',
+    port: 8883
+  };
+  var client = mqtt.connect('mqtts:mqtt.ably.io', options);
+  client.on('connect', () => {
+    client.subscribe('[?rewind=1]{{RANDOM_CHANNEL_NAME}}');
+  });
+  client.on('message', (topic, message) => {
+    ...
```

Comment:
In other code samples we console log the message as opposed to have `...` code

Reply:
@QuintinWillison can you help complete this code sample?

Reply:
See https://github.com/ably/docs/pull/880#issuecomment-637452883

Reply:
@MarkWoulfeAbly once this is all done, please grab me so that we can talk about getting you up to speed on running code samples locally so that for JS (typically our base language), you can write and test code samples.